modules/cas_cache/volume/vol_block_dev_top.c

```diff
@@ -572,8 +635,7 @@ int kcas_core_activate_exported_object(ocf_core_t core)
 	int result;
 
 	result = kcas_volume_activate_exported_object(volume,
-			&_blockdev_exp_obj_ops);
-
+			&kcas_core_exp_obj_ops);
 	if (result) {
 		printk(KERN_ERR "Cannot activate exported object, %s.%s. "
```

Comment:
It contains this information anyway, as it's parametrized with cache name and core name, so it would look like:
`Cannot activate exported object, cache1.core1. Error code -5`